src/controllers/update/update.signed.out.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    return res.render(config.UPDATE_SIGNED_OUT_PAGE, {
+      templateName: config.UPDATE_SIGNED_OUT_PAGE,
+      hideBanners: true
```

Comment:
Are we the only renderer of this page so do we need to pass in hideBanners or just do it in view?

Reply:
might result in most of the template copied.